src/lib/InstantSearch.ts

```diff
@@ -599,11 +625,14 @@ See ${createDocumentationLink({
     }
   });
 
-  public scheduleRender = defer(() => {
+  public scheduleRender = defer((resetsStatus: boolean = true) => {
```

Comment:
I don't see why scheduleRender would ever be in user code, as well as it not being documented

Reply:
That's good news, only asking because I know we sometimes share code snippets that leverage semi-internals as workarounds.

Reply:
oh and the behaviour if you don't pass the parameter hasn't changed, as it defaults to the same behaviour as before

Reply:
Yes, I saw that but I was thinking that if we ever need more arguments, positional might become a hassle and we'll want to move to a single object. But right now an object would be more code.

If we feel fine changing that because it's not in the contract, I'm good.